server/test/accountCreation.test.js

```diff
@@ -0,0 +1,34 @@
+const accountCreationHandler = require('../models/handlers/accountCreation');
+
+ describe('Test to check valid account creation', () => {
+     beforeEach(() => {
+         jest.resetModules();
+     });
+//Create a test using Arrange, Act, Assert format
+     test('Checking if the account creation details are correct', async() => {
```

Comment:
I would indicate initial setup in the test description to avoid confusion. As per the team's code terms, initial-setup tests involve testing the stub, i.e., fixed values returned from a method rather than the logic of that particular method/functionality.

Reply:
Hi Amy,
Thank you for suggesting that!
**My action**: I've clarified that this test is an initial set up for testing a stub. 
You can now see it in line 8 of server/test/accountCreation.test.js

Reply:
Yes it does. However the automated tests are failing at the moment. Please ensure that the imported file name is also updated in the `accountCreation.test.js` file.

Reply:
Hi Amy, which line in this file are you refering to? I can't seem to locate what is making this break.

Reply:
Thanks for clarifying Amy! I have made this change in the most recent commit. I'd like to know if that fixes the issue?

Reply:
Hi Nancy, the tests are still failing (photo attached). Remember to export the proper methods in your files.
![image](https://user-images.githubusercontent.com/32720322/96777759-bcf4c480-13b8-11eb-877b-57ac8d606357.png)

Reply:
Hi Amy, thanks for providing the screenshot! I see the issue: I mislabelled the export function as createAccountUser and the function name elsewhere was createUserAccount. I have made the names consistent in the export line and the function name. Does the most recent commit fix this issue?

Reply:
Yes, that is perfect. I appreciate the quick changes!